app-backend/database/src/main/scala/com/azavea/rf/database/tables/Buckets.scala

```diff
@@ -239,4 +256,37 @@ object Buckets extends TableQuery(tag => new Buckets(tag)) with LazyLogging {
       sceneBucketJoinQuery.delete
     }
   }
+
+  /** Removes scenes from bucket
+    *
+    * @param sceneIds Seq[UUID] primary key of scenes to remove from bucket
+    * @param bucketId UUID primary key of bucket these scenes will be removed from
+    */
+  def deleteScenesFromBucket(sceneIds: Seq[UUID], bucketId: UUID)
+    (implicit database: DB): Future[Int] = {
+
+    val sceneBucketJoinQuery = for {
+      s <- ScenesToBuckets if s.sceneId.inSet(sceneIds) && s.bucketId === bucketId
+    } yield s
+
+    database.db.run {
+      sceneBucketJoinQuery.delete
+    }
+  }
+
+  /** Removes all scenes from bucket
+    *
+    * @param bucketId UUID primary key of bucket to remove all scenes from
+    */
+  def deleteAllScenesFromBucket(bucketId: UUID)
```

Comment:
We're not really using this, and I don't like having unused code around. If I don't find a use for this by the end of this PR, I'll cut it.